decidim-proposals/db/migrate/20180413135249_fix_nil_threshold_per_proposal.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class FixNilThresholdPerProposal < ActiveRecord::Migration[5.1]
+  class Component < ApplicationRecord
+    self.table_name = :decidim_components
+  end
+
+  def change
+    proposal_components = Component.where(manifest_name: "proposals")
+
+    proposal_components.select do |component|
```

Comment:
I guess this equally works with `select`, but plain `each` looks more correct since you don't need to select anything.

Reply:
I forgot to change it to an `each` 😅